### PR TITLE
Db context

### PR DESCRIPTION
**Pull request by viniVendrame**
Created DevContext, DbSet properties and their Tables.

### CODE REVIEW

**Review by caiomateusreis — APPROVED** [review]

**Review by sr-adilson — APPROVED** [review]
Ok

File: Developers/Data/Context/DevContext.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+using System;
+using System.Collections.Generic;
+using System.Data.Entity;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Data.Context
+{
+    class DevContext : DbContext
```
Comment (felipe-hofelmann) [inline Developers/Data/Context/DevContext.cs]: Missing a public in the DevContext class

**Review by felipe-hofelmann — CHANGES_REQUESTED** [review]
Missing a public in the DevContext class

**Review by felipe-hofelmann — APPROVED** [review]
show

**Pull request merged**